src/protocol/RemotingCommand.cpp

```diff
@@ -50,6 +50,46 @@ RemotingCommand::RemotingCommand(int code, string language, int version,
       m_remark(remark),
       m_pExtHeader(pExtHeader) {}
 
+RemotingCommand::RemotingCommand(const RemotingCommand& command) {
```

Comment:
```c++
 RemotingCommand::RemotingCommand(const RemotingCommand& command) 
  : m_code(command. m_code) 
  ...
  {
```

Reply:
there are too many the members, so no need use this style